src/DropZone.xml

```diff
@@ -65,36 +66,65 @@
-            <description>Microflow is called when an uploaded file is removed.</description>
+            <description>Nanoflow is called when file is dropped.</description>
             <returnType type="Boolean"/>
         </property>
-        <property key="onUploadMicroflow" type="microflow" required="false" entityProperty="fileEntity">
-            <caption>On upload</caption>
+
+
+        <property key="onRemoveEvent" type="enumeration" defaultValue="doNothing">
+            <caption>On remove of a file</caption>
             <category>Events</category>
-            <description>Microflow is called when a file is uploaded.</description>
-            <returnType type="Boolean"/>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="doNothing">Do nothing</enumerationValue>
+                <enumerationValue key="callMicroflow">Call a microflow</enumerationValue>
+                <enumerationValue key="callNanoflow">Call a nanoflow</enumerationValue>
+            </enumerationValues>
         </property>
-        <property key="onDropNanoflow" type="nanoflow" required="false" entityProperty="fileEntity">
-            <caption>On drop nanoflow</caption>
+
+        <property key="onRemoveMicroflow" type="microflow" required="false" entityProperty="fileEntity">
+            <caption>On remove microflow</caption>
             <category>Events</category>
-            <description>Nanoflow is called when file is dropped.</description>
+            <description>Microflow is called when an uploaded file is removed.</description>
             <returnType type="Boolean"/>
         </property>
-        <property key="onRemoveNanoflow" type="nanoflow" required="false" entityProperty="fileEntity">
+
+         <property key="onRemoveNanoflow" type="nanoflow" required="false" entityProperty="fileEntity">
             <caption>On remove nanoflow</caption>
             <category>Events</category>
             <description>Nanoflow is called when an uploaded file is removed.</description>
             <returnType type="Boolean"/>
         </property>
+
+        <property key="onUploadEvent" type="enumeration" defaultValue="doNothing">
+            <caption>On upload of a file</caption>
+            <category>Events</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="doNothing">Do nothing</enumerationValue>
+                <enumerationValue key="callMicroflow">Call a microflow</enumerationValue>
+                <enumerationValue key="callNanoflow">Call a nanoflow</enumerationValue>
+            </enumerationValues>
+        </property>
+
+        <property key="onUploadMicroflow" type="microflow" required="false" entityProperty="fileEntity">
+            <caption>On upload microflow</caption>
+            <category>Events</category>
+            <description>Microflow is called when a file is uploaded.</description>
```

Comment:
Remove full stop at every end of the description and
change to `Microflow that will be executed when a file is uploaded`